certbot/plugins/manual.py

```diff
@@ -191,16 +143,9 @@ def _perform_achall_with_script(self, achall):
             env['CERTBOT_TOKEN'] = achall.chall.encode('token')
         else:
             os.environ.pop('CERTBOT_TOKEN', None)
-        if isinstance(achall.chall, challenges.TLSSNI01):
-            env['CERTBOT_CERT_PATH'] = self.tls_sni_01.get_cert_path(achall)
-            env['CERTBOT_KEY_PATH'] = self.tls_sni_01.get_key_path(achall)
-            env['CERTBOT_SNI_DOMAIN'] = self.tls_sni_01.get_z_domain(achall)
-            os.environ.pop('CERTBOT_VALIDATION', None)
-            env.pop('CERTBOT_VALIDATION')
-        else:
-            os.environ.pop('CERTBOT_CERT_PATH', None)
-            os.environ.pop('CERTBOT_KEY_PATH', None)
-            os.environ.pop('CERTBOT_SNI_DOMAIN', None)
+        os.environ.pop('CERTBOT_CERT_PATH', None)
+        os.environ.pop('CERTBOT_KEY_PATH', None)
+        os.environ.pop('CERTBOT_SNI_DOMAIN', None)
```

Comment:
I think we can remove unsetting these environment variables as we only set them in the now deleted code for TLS-SNI-01.

Reply:
Done.